pkg/chartutil/load.go

```diff
@@ -135,6 +135,9 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 			return c, errors.New("values.toml is illegal as of 2.0.0-alpha.2")
 		} else if f.Name == "values.yaml" {
 			c.Values = &chart.Config{Raw: string(f.Data)}
+		} else if f.Name == "secrets.yaml" {
+			c.Secrets = &chart.Config{Raw: string(f.Data)}
+			c.Secrets.Type = chart.ConfType_SECRETS
```

Comment:
You don't have to because the default value is zeroed out, meaning it'll default to ConfType_VALUES (per the go spec)